docs/hugo/content/guide/annotations.md

```diff
@@ -21,6 +21,16 @@ it is _not_ deleted in Azure. In REST API terminology, PUT and GET are allowed w
     
 Unknown values default to `manage`.
 
+### `serviceoperator.azure.com/credential-from`
+
+Instructs the operator to use a particular credential for the resource. This credential supersedes any global or
+namespace scoped credentials the operator has configured.
+
+Allow values are:
+- The name of any secret in the same namespace as the resource
```

Comment:
Not necessarily should be in the same namespace, they can use the secret in another namespace as well using `Namespace/resourceName` syntax as well

Reply:
If its in the same namespace, then they can only use the credential name without the namespaceName

Reply:
I think these docs are correct - don't we explicitly prohibit cross namespace secrets?
Maybe that needs to be called out here too.

Reply:
We just changed this. We used to technically (undocumented) support cross-namespace. I removed that in #2919